src/notarytool.ts

```diff
@@ -83,9 +83,14 @@ export async function notarizeAndWaitForNotaryTool(opts: NotaryToolStartOptions)
     ];
 
     const result = await spawn('xcrun', notarizeArgs);
-    const parsed = JSON.parse(result.output.trim());
+    let parsed: any;
+    try {
+        parsed = JSON.parse(result.output.trim());
+    } catch (err) {
+        throw new Error(`Failed to notarize via notarytool\n\nCould not parse output as JSON\n\n${result.output.trim()}`);
+    }
 
-    if (result.code !== 0 || !parsed.status || parsed.status !== 'Accepted') {
+    if (!parsed.status || parsed.status !== 'Accepted') {
```

Comment:
Hmmmm, the errors I was encountering when `result.code !== 0` the JSON.stringify would error anyway, but yes that sounds more robust.

Also a small pain-point: While I was building my app I would often have to run the `notarytool log` command as well.  I think it's worth bringing these logs up out from behind the debug flag.  I'll add that now and can revert if you think otherwise :)

Reply:
I also tweaked the message to say `Failed with unexpected result:` instead of `Could not parse output as JSON` as I think it makes more sense to the end user (they won't know we need to parse the error json to get the `notarytool log` diagnostics).